deploy/examples/create-external-cluster-resources.py

```diff
@@ -986,6 +997,10 @@ def get_caps_and_entity(self, user_name):
             if "client.csi-rbd-node" != user_name:
                 self._arg_parser.restricted_auth_permission = True
             return self.get_rbd_node_caps_and_entity()
+        if "client.healthchecker" in user_name:
+            if "client.healthchecker" != user_name:
```

Comment:
we can skip this check because I believe we only have one operator even if we are connected with multiple clusters

Reply:
okay 👍🏽

Reply:
I think, we can keep the check, if remove the caps, and the method was called to get the csi user caps and entity then also healthchecker user caps and entity will be retured

Reply:
I was talking about the `if "client.healthchecker" != user_name:`

Reply:
removed

Reply:
I guess the changes are not commited, can you re-look